lib/src/users.rs

```diff
@@ -170,7 +175,7 @@ mod tests {
         let (users, _temp) = new_test();
 
         assert_eq!(None, users.slack_user_name("joe"));
-        assert_eq!(None, users.slack_direct_message("joe"));
+        assert_eq!(None, users.slack_direct_message("joe", &"repo".into()));
```

Comment:
nit: i think should update tests to say `org/repo` to demonstrate better what it looks like for real.

Reply:
Done